src/MessagingAttachment/MessagingAttachment.less

```diff
@@ -162,6 +161,72 @@
   }
 }
 
+.MessagingAttachment--ParentContainer.MessagingAttachment--ParentContainer--IsUpload {
+  overflow: visible; // ensures you can see the x icon in top right corner
+}
+
+// Family portal specific styling
+.MessagingAttachment--ParentContainer.MessagingAttachment--FamilyPortal {
+  border-radius: @size_m;
+  background-color: @neutral_white;
+  box-shadow: @size_none @size_3xs 0.375rem rgba(71, 76, 94, 0.3);
+  .boxShadow--medium();
+  margin-top: @size_2xs;
+  color: @neutral_dark_gray;
+
+  .MessagingAttachment--Container {
+    border: unset;
+    border-radius: unset;
```

Comment:
Similar to my earlier comment, can we get by with a `border: 0` and `border-radius: 0` here?